documentation/src/main/docs/kafka/health.md

```diff
@@ -1,54 +1,39 @@
 # Health reporting
 
-The Kafka connector reports the readiness and liveness of each channel
+The Kafka connector reports the startup, readiness and liveness of each channel
 managed by the connector.
 
 !!!note
     To disable health reporting, set the `health-enabled` attribute for the
     channel to `false`.
 
-## Readiness
+## Startup & Readiness
```

Comment:
While I love walls of text, I believe that a summary in a table would be good to have. WDYT?